### ISSUE TITLE
quartz 2.3.0

### ISSUE THREAD

**Issue by beetleman** [prepor/twarc#12]
There is any well known blocker for updating `quartz` to newest version? I can start working on it because in my current project we plan to use `twarc` because is quite simple to use and source code is not so complicated compared to other quartz wrappers.

**Comment by prepor** [comment]
No, there is no. I'm not using quartz these days, but if anyone tests lib against a newer version I would love to bump version in the library.

**Comment by beetleman** [comment]
I started some work there https://github.com/beetleman/twarc/tree/feature/upgrade-quartz-to-2.3.2-and-other-deps-to-newest-versions. I added `docker-compose.yml` file for make easier to run tests against PostgreSQL (I need persistent scheduler so its important for me to tests it). I will add deps.edn too so I can use it in project without messing with clojars. In ongoing week I plan to integrate `twarc` to project so I will test it in real life scenario so on the end of week I should have PR for you:D

**Comment by prepor** [comment]
@beetleman awesome, thank you!

**Comment by beetleman** [comment]
It working in our stage env so its should be ok, I will test it more during this week so on next weekend I should be ready to publish PR with it.

**Comment by konradszydlo** [comment]
@beetleman any luck with the PR?

**Comment by beetleman** [comment]
W using it production few weeks without any problems so I can provide PR, but we using it with in memory storage now. I will try prepare PR today . But as I said I do not test it extensively with PostgreSQL or any other db:/.
@konradszydlo  if you want use it right now you can do it:

```clojure
twarc                            {:mvn/version "0.1.12"
                                   :exclusions
                                   [org.quartz-scheduler/quartz
                                    org.quartz-scheduler/quartz-jobs]}
 org.quartz-scheduler/quartz      #:mvn{:version "2.3.2"}
 org.quartz-scheduler/quartz-jobs #:mvn{:version "2.3.2"}

```
this example is for `deps.edn` but it will be similar to lein.

**Comment by beetleman** [comment]
#13 and travis status for it: https://travis-ci.com/github/beetleman/twarc

**Comment by prepor** [comment]
@beetleman thank you very much for contribution! I merged it and released as version "0.1.13"

**Comment by beetleman** [comment]
https://clojars.org/twarc/versions/0.1.13 :tada:
